### DISCUSSION TITLE
Where can I download the latest icub bill of materials？

### DISCUSSION THREAD

**Discussion by xilihutu888** [robotology/community#482] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20624124/107928162-f3330100-6fb2-11eb-97b2-a4a0e9e2e51f.png)
Where can I download the latest icub bill of materials？

**Comment by fiorisi** [comment, marked as answer]
Dear @nan2854, for the BOM you can refer to [this comment](https://github.com/robotology/community/discussions/477#discussioncomment-298846). Using PTC Creo you can easily generate the BOM of the iCub assembly.

**Reply by xilihutu888** [reply]
Thank you for your reply. I opened PTC Creo but some parts are missing. Could you please help me export a BOM for me to download? Thank you very much.

**Reply by fiorisi** [reply]
> I opened PTC Creo but some parts are missing.

It should work. Which parts are missing?

**Reply by xilihutu888** [reply]
![20210218224610](https://user-images.githubusercontent.com/20624124/108373901-5b941380-723b-11eb-878a-c825b347670e.png)
![20210218224620](https://user-images.githubusercontent.com/20624124/108373906-5cc54080-723b-11eb-9772-b3fc8acd811b.png)
![20210218224631](https://user-images.githubusercontent.com/20624124/108373918-5df66d80-723b-11eb-9726-bda6639341e3.png)
![20210218224636](https://user-images.githubusercontent.com/20624124/108373924-5f279a80-723b-11eb-9eb8-9f22a97241e0.png)

Unfortunately, I don’t have permission to export bom. Please help me and provide me with a copy of the latest and complete bom to download. Thank you for your reply.

**Reply by fiorisi** [reply]
@nan2854 I've tried to open the assembly as described [here](https://github.com/robotology/community/discussions/477#discussioncomment-298846). I don't see any issue, did you configure `cad-libraries` correctly?
![image](https://user-images.githubusercontent.com/8970230/108382036-b2ccc080-7208-11eb-9991-7450571451c2.png)

![image](https://user-images.githubusercontent.com/8970230/108382089-be1fec00-7208-11eb-8103-c432a21cd00b.png)

**Reply by xilihutu888** [reply]
![20210219231332](https://user-images.githubusercontent.com/20624124/108522832-2c97a380-7308-11eb-98ad-b0fe70f1aa8d.png)

![20210219231225](https://user-images.githubusercontent.com/20624124/108522855-33beb180-7308-11eb-8090-eecb900c9f15.png)
![20210219231209](https://user-images.githubusercontent.com/20624124/108522880-3caf8300-7308-11eb-9d56-ad85898f5a27.png)
Why are there so many files missing from the (cad-libraries) repository I cloned? Please help me. How to solve it? Thank you for your reply

**Reply by fiorisi** [reply]
Dear @nan2854, recently we made public the repository [`icub-snapshot`](https://github.com/icub-tech-iit/icub-snapshots). Here you can find the complete CAD model of the latest robot version `2.6`.

**Reply by xilihutu888** [reply]
Oh my goodness, this is really exciting, thank you for your reply, thank you for all the contributions of the iCub robot team, I will be able to start making iCub robots, can I apply to join the iCub robot team? I am a robot fanatic, even if I am an external editor or volunteer.
